docs/plugins/survey.md

```diff
@@ -42,169 +28,203 @@ Other parameters can be left unspecified if the default value is acceptable.
 
-Present a question with a limited set of options. The participant can only select one option.
+- [Expression example](https://surveyjs.io/form-library/examples/expression-question-for-dynamic-form-calculations/jquery) 
+- [Expression API documentation](https://surveyjs.io/form-library/documentation/api-reference/expression-model)
 
-In addition to the parameters for all question types, the multi-choice question type also offers the following parameters. 
-Parameters with a default value of *undefined* must be specified.
-Other parameters can be left unspecified if the default value is acceptable.
+#### file
 
-Parameter | Type | Default Value | Description
-----------|------|---------------|------------
-options | array of strings | *undefined* | This array contains the set of multiple choice options to display for the question.
-option_reorder | string | "none" | One of: "none", "asc", "desc", "random". If "none", the options will be listed in the order given in the `options` array. If `random`, the option order will be randomized. If "asc" or "desc", the options will be presented in ascending or descending order, respectively.
-columns | integer | 1 | Number of columns to use for displaying the options. If 1 (default), the choices will be displayed in a single column (vertically). If 0, choices will be displayed in a single row (horizontally). Any value greater than 1 can be used to display options in multiple columns. 
-correct_response | string | null | String from the `options` array that should be considered correct. If specified, the data will include a `correct` property that indicates whether the response was correct (`true`) or not (`false`).
+This question allows participants to upload one or more files (images, documents, etc.). It provides drag-and-drop and browse/select options. Files can be uploaded to a server or stored directly in the survey results JSON object as base64-encoded text.
 
-#### Multi-select
+!!! note 
+    When using this question type, you will need to choose how to handle the files. One option is to save the file with the rest of the response data as a Base64-encoded text string (`storeDataAsText: true`), but keep in mind that this will significantly increase the size of the response data and should only be used for small files. Another option is to upload the participant's file to a server, which you can do using the `onUploadFiles` event handler in the jsPsych `survey_function` function. You can read more about these options in the SurveyJS [file upload documentation](https://surveyjs.io/form-library/examples/file-upload/jquery#content-docs), and see an example of the file upload option in the `index.js` part of this [code example](https://surveyjs.io/form-library/examples/file-upload/jquery#content-code).
```

Comment:
Not necessary, but would be neat to add example of using DataPipe's base64 file save feature in the future.

Reply:
Yeah, absolutely -- no need to do that now.